app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/cohort/CohortPixelInterceptor.kt

```diff
@@ -68,10 +70,12 @@ class CohortPixelInterceptor @Inject constructor(
     }
 
     companion object {
-        private const val COHORT_PARAM = "atp_cohort"
+        @VisibleForTesting
+        const val COHORT_PARAM = "atp_cohort"
```

Comment:
You can probably make this `internal` to limit the visibility

```suggestion
        internal const val COHORT_PARAM = "atp_cohort"
```